microsoft-edge/dev/Dockerfile

```diff
@@ -50,7 +50,7 @@ RUN set -eux; \
 	} | tee /etc/default/microsoft-edge-dev
 
 # https://packages.microsoft.com/repos/edge/pool/main/m/microsoft-edge-dev/
-ENV EDGE_VERSION 120.0.2210.1-1
+ENV EDGE_VERSION 120.0.2186.2-1
```

Comment:
🤬 what is going on?? (https://feedbackportal.microsoft.com/feedback/idea/c635132e-a979-ee11-a81c-0022484e5453)

Reply:
Aha, 120.0.2210.4-1 is up now -- let's give it a go, I guess!